### PR TITLE
Math components

### PR DESCRIPTION
**Pull request by ger619**
In the previous project, you worked on the setup of a React application using Create React App (CRA), now you should:
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components (All components should be class-based).
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,10 +2,13 @@ import React from 'react';
...
+// eslint-disable-next-line import/no-named-as-default
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
+    <Calculator />
   </React.StrictMode>,
 );
```
Comment (KDlamini) [inline src/index.js]: In React it is best practice to ONLY render the `App` component on the React DOM. Kindly remove the `Calculator` component from the `index.js` file.

File: src/App.js [hunk 2]
```diff
@@ -1,33 +1,10 @@
...
-        {' '}
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends Component {
+  render() {
+    return <div className="app" />;
+  }
 }
```
Comment (KDlamini) [inline src/App.js]: The `Calculator` component should be rendered in the `App` component. In the future, you will mostly route all parent components in the `App` component. The `App` component serves as the main structure for the application that contains the Navigation and different pages hosted by the application.

You should have something like this:
```
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @ger619 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-labana — APPROVED** [review]
# **Approved ✔️**

Hi @ger619

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**